com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -258,22 +263,37 @@ internal void EndSceneEvent(uint sceneEventId)
         /// <summary>
         /// Gets the scene name from full path to the scene
         /// </summary>
+        /// <returns>scene name</returns>
         internal string GetSceneNameFromPath(string scenePath)
         {
             var begin = scenePath.LastIndexOf("/", StringComparison.Ordinal) + 1;
             var end = scenePath.LastIndexOf(".", StringComparison.Ordinal);
             return scenePath.Substring(begin, end - begin);
         }
 
+        /// <summary>
+        /// Returns the full path of the scene and scene name as it
+        /// is displayed in the Build Settings Scenes in Build list.
+        /// </summary>
+        /// <param name="scenePath"></param>
+        /// <returns>full path and scene name</returns>
+        internal string GetScenePathAndName(string scenePath)
```

Comment:
Please conspicuously mark this and similar string parsing trickery with "to be removed with scene v2.0, etc"

Reply:
Done!